metc-wastewater-covid-monitor/wastewater-monitoring-report.Rmd

```diff
@@ -0,0 +1,564 @@
+---
```

Comment:
Should this be in the metc-wastewater-covid-monitor/ directory?

Reply:
i agree! this should be in its own directory.